src/bundle/Resources/views/themes/admin/ui/menu/main_2nd_level.html.twig

```diff
@@ -87,9 +98,10 @@
     {% endif %}
 
     {% set link_attributes = item.linkAttributes|merge(linkAttributes) %}
+    {% set group_name_class = renderItemAsGroup is defined and renderItemAsGroup ? 'ibexa-main-menu__item-text-group' %}
 
     <a{{ knp_menu.attributes(link_attributes) }}>
```

Comment:
If this will be a group label should this be still an `<a>` tag? I think it shouldn't be a link

Reply:
It can still lead to a valid route, although it usually doesn't (otherwise it's difficult to create a proper accordion with open/close).

IMO do not render it as an `<a>` element, regardless of route - at least for now.